clients/cloud/rust/src/consumer.rs

```diff
@@ -0,0 +1,57 @@
+extern crate futures;
```

Comment:
add copyright blob

Reply:
do you mean a confluent one that we have in the other examples?